app/src/main/java/org/simple/clinic/bloodsugar/unitselection/BloodSugarUnitSelectionDialog.kt

```diff
@@ -132,6 +153,15 @@ class BloodSugarUnitSelectionDialog : AppCompatDialogFragment(), BloodSugarUnitS
     dismiss()
   }
 
+  override fun prefilledBloodSugarUnitSelection(blodSugarUnitPreference: BloodSugarUnitPreference) {
+    val bloodSugarUnitPreferenceButton: RadioButton? = when (blodSugarUnitPreference) {
```

Comment:
This does not need to be nullable, right? If the value is anything other than `Mg` or `Mmol` (which is impossible since it's an enum), this will not be a valid state and we can raise an error instead.